lib/checkexceptionsafety.cpp

```diff
@@ -251,6 +259,19 @@ static const Token * functionThrowsRecursive(const Function * function, std::set
 
     for (const Token *tok = function->functionScope->bodyStart->next();
          tok != function->functionScope->bodyEnd; tok = tok->next()) {
+        if (Token::simpleMatch(tok, "if")) {
+            tok = tok->astParent();
+            const Token *condTok = tok->astOperand2();
+            // check if condition is possibly false, since template arguments are not set to known
+            if (isPossibleBool(condTok, false)) {
+                tok = tok->link()->next();
+                if (Token::simpleMatch(tok, "{"))
+                    tok = tok->link();
+                else while (tok && !Token::simpleMatch(tok, ";"))
+                        tok = tok->next();
```

Comment:
this does not seem to be formatted properly. we have the `runformat` script in the cppcheck folder could you run that..

Reply:
Hm, this seems to be what uncrustify wants. But I could add braces to make it more readable.